File: C++/Daniel Cangrejo/Geometria/lineMidPoint.cpp
#include<bits/stdc++.h>
#include<cstdlib>
using namespace std;
string midPoint(int x1,int y1,int x2,int y2){
    return (to_string((x1+x2)/2)+","+to_string((y1+y2)/2));
}
int main() {
    int x1,y1;
    int x2,y2;
    cin>>x1>>y1>>x2>>y2;
    string medio=midPoint(x1,y1,x2,y2);
    cout<<medio<<endl;
}

File: Codigos estudiantes por lenguaje/Java/Cristian Maldonado/Geometria/Line_Mid_Point.java
public class Line_Mid_Point {

    static void midpoint(int x1, int x2,
            int y1, int y2) {
        System.out.print((x1 + x2) / 2
                + " , " + (y1 + y2) / 2);
        System.out.println("");
    }

    public static void main(String[] args) {
        int x1 = -1, y1 = 2;
        int x2 = 3, y2 = -6;
        midpoint(x1, x2, y1, y2);
    }
}
